zebra-state/src/request.rs

```diff
@@ -495,8 +494,9 @@ pub enum Request {
     ///
     /// # Note
     ///
-    /// Finalized and non-finalized blocks are an internal Zebra implementation detail.
-    /// There is no difference between these blocks on the network, or in Zebra's
+    /// `SemanticallyVerifiedBlock`, `ContextuallyVerifiedBlock` and
+    /// `CheckpointVerifiedBlock` are an internal Zebra implementation detail.
```

Comment:
If we use doc links here, `rustdoc` will tell us if we have typos or missed renames:
```suggestion
    /// [`SemanticallyVerifiedBlock`], [`ContextuallyVerifiedBlock`] and
    /// [`CheckpointVerifiedBlock`] are an internal Zebra implementation detail.
```